tests/performance-jmh/pom.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <parent>
+        <groupId>org.apache.activemq.tests</groupId>
+        <artifactId>artemis-tests-pom</artifactId>
+        <version>2.17.0-SNAPSHOT</version>
```

Comment:
Since this module isn't part of the main build, this presumably wont be updated during the release process, meaning it wont be possible to build the module from source once released (and the matching snapshot is deleted) without modification or also building the older snapshot. That is, assuming the test source actually compiles at all at that point, which it may not if it hasn't been used in a bit. Not necessarily a blocker, but something to note.

(Handling this is part of what the profile trickery in the proton-j source inspiration does, ensuring the module is enabled and re-versioned during release process, such that it can actually be used later from the produced source release, and keeps a check that the module actually compiles).

Reply:
You have to make sure the source build would not include it.

Reply:
You really dont. You can decide to.

Reply:
@gemmellr We had people making a big deal about the extra-tests as part of the source tree years ago.

perhaps we didn't need that removed?